storage/azure/src/client/AzureClientStorage.ts

```diff
@@ -2,32 +2,32 @@
  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
-import { createReadStream, promises } from "fs";
+import { promises } from "fs";
 import { dirname } from "path";
 import { Readable } from "stream";
 
 import { inject, injectable } from "inversify";
 
+import { ClientStorage } from "@itwin/object-storage-core/lib/client";
 import {
-  assertFileNotEmpty,
   assertRelativeDirectory,
-  ClientStorage,
-  FrontendTransferData,
-  isLocalUrlTransfer,
+  Types,
+} from "@itwin/object-storage-core/lib/common";
```

Comment:
I don't like these explicit `lib/common` imports, why not import these from the index file?

Reply:
done, see comment below